pkg/vectorstores/chromadb/chroma.go

```diff
@@ -107,6 +110,10 @@ func (s Store) AddDocuments(ctx context.Context, docs []schema.Document, options
 		return ErrEmbedderWrongNumberVectors
 	}
 
+	embedTime := time.Since(startTime) / time.Millisecond
+
+	fmt.Printf("embed time cost: %dms\n", embedTime)
```

Comment:
please sync code to the latest commit which we have deprecated this vector store from kubeagi to langchaingo.